src/components/SearchBar.tsx

```diff
@@ -0,0 +1,69 @@
+import { View, StyleSheet, TextInput } from 'react-native';
+import React from 'react';
+
+type SearchBarProps = {
+  setSearchValue: (text: string) => void;
+  searchValue: string;
+  membersData: MembersDataProps[];
+  setMembersData: ([]) => void;
+};
+
+type MembersDataProps = {
+  github_display_name: string;
+};
+
+const SearchBar = ({
+  setSearchValue,
+  searchValue,
+  membersData,
+  setMembersData,
+}: SearchBarProps) => {
+  const searchFunction = (text: string) => {
+    setSearchValue(text);
+    const updatedData = text
+      ? membersData?.filter((member) =>
+          member?.github_display_name
+            ?.toLowerCase()
+            .includes(text.toLowerCase()),
+        )
+      : membersData;
+      setMembersData(updatedData)
+  };
+
+  return (
+    <View style={styles.container}>
+      <TextInput
+        placeholder="Search"
+        style={styles.formField}
+        placeholderTextColor={'#888888'}
```

Comment:
Can you create the variable for this jex colour and why we not using style from style files ?

Reply:
I think you mean Hex color!
won't be treated as a priority task because we will be moving to the design theme which Ankush created. will be doing it in a separate ticket. 

discussed !!!!